zmon_worker_monitor/builtins/plugins/scalyr.py

```diff
@@ -49,6 +50,36 @@ def __init__(self, read_key, scalyr_region=None):
     def count(self, query, minutes=5):
         return self.timeseries(query, function='count', minutes=minutes, buckets=1, prio='low')
 
+    def logs(self, query, max_count=100, minutes=5, continuation_token=None):
+
+        if not query or not query.strip():
+            return {'messages': [], 'continuation_token': None}
```

Comment:
I think it would be better to raise exception instead (e.g. ``CheckError``) with error details. That way we point the user to the source of the problem and they can fix it. Returning empty list could be misleading to users.